Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h

```diff
@@ -27,6 +27,8 @@
 #include "ITStracking/Road.h"
 #include "ITStracking/Tracklet.h"
 
+#include "SimulationDataFormat/MCCompLabel.h"
```

Comment:
This breaks CUDA compilation, by pulling in MCCompLabel to GPU compiled file.

Reply:
uff, sorry!

Reply:
In the long run, I want to make the MC container GPU compatible, but we have to do something about the gsl::span. Should discuss this when @sawenzel is back.

Reply:
No issue